scripts/deploy.sh

```diff
@@ -55,7 +55,7 @@ function update_tagged_files {
 
 function deploy_versioned_files {
     local version=$1
-    local in_files='../dist/Chart*.js'
+    local in_files='../dist/chart*.js'
```

Comment:
Is this case change something we'd need to mention in the migration docs?

Reply:
Yeah, and we need to discuss if even changing the file name is worth the trouble.

Reply:
@benmccann thoughts on this? @simonbrunel this was on the list of things you had written down for v3.

Reply:
I feel it's more standard to use lowercase and will be consistent with plugins. v3 is a good opportunity to add that one. And since these files are referenced from `package.json`, in most case it should not require users any change. But will still need to be mentioned in the migration guide.

Reply:
I'm confused. the package.json references uppercase, the rollup build uppercase but this scripts assumes it is lowercase? or do I get the script wrong?

Reply:
You are right, I didn't review the whole PR. "jsdelivr", "unpkg", "main" and "module" in `package.json` need to be changed to use the lowercase namee. Same for rollup.

Reply:
I don't have strong feelings one way or the other. I'm fine with whatever the consensus is

Reply:
I created a new PR for the case. This one has still some issues, I'll change it to ready when I'm happy with it.